### ISSUE TITLE
Failed while training start from scratch

### ISSUE THREAD

**Issue by tkianai** [ZhaoJ9014/face.evoLVe.PyTorch#101]
I use the preprocessed data provided by author, and training with the default config start from scratch, as for there is no `IR_SE_50` backbone checkpoint provided. But the training loss comes to `nan` after one or several epochs(under five), Is it impossible to train from scratch? Any successful experiences?
